test/LumpedRateModelWithoutPores.cpp

```diff
@@ -16,85 +16,124 @@
+	const std::vector<double> absTol = { 1e-12, 1e-12, 1e-12, 1e-12 };
+	const std::vector<double> relTol = { 1.0, 1.0, 1.0, 1.0 };
+	cadet::test::column::testEOCReferenceBenchmark(modelFilePath, refFilePath, convFilePath, "001", absTol, relTol, 4, 16, 0, true);
 }
 
-TEST_CASE("LRM forward sensitivity forward vs backward flow", "[LRM],[Sensitivity],[Simulation]")
+TEST_CASE("LRM numerical EOC Benchmark with parameter sensitivities for SMA LWE case", "[releaseCI],[EOC],[EOC_LRM_FV]")
 {
-	const double absTols[] = {500.0, 8e-7, 9e-7, 2e-3};
-	const double relTols[] = {7e-3, 5e-5, 5e-5, 9e-4};
-	const double passRatio[] = {0.99, 0.97, 0.97, 0.98};
-	cadet::test::column::testFwdSensSolutionForwardBackward("LUMPED_RATE_MODEL_WITHOUT_PORES", absTols, relTols, passRatio);
+	const std::string& modelFilePath = std::string("/data/model_LRM_reqSMA_4comp_benchmark1.json");
+	const std::string& refFilePath = std::string("/data/ref_LRM_reqSMA_4comp_sensbenchmark1_FV_Z4096.h5");
+	const std::string& convFilePath = std::string("/data/convergence_LRM_reqSMA_4comp_sensbenchmark1.json");
+	const std::vector<double> absTol = { 1e-12, 1e-12, 1e-12, 1e-12 };
+	const std::vector<double> relTol = { 1.0, 1.0, 1.0, 1.0 };
+	cadet::test::column::testEOCReferenceBenchmark(modelFilePath, refFilePath, convFilePath, "000", absTol, relTol, 2, 8, 0, true);
 }
 
-TEST_CASE("LRM consistent initialization with linear binding", "[LRM],[ConsistentInit]")
+TEST_CASE("LRM time derivative Jacobian vs FD", "[LRM],[UnitOp],[Residual],[Jacobian],[CI],[FD]")
 {
-	cadet::test::column::testConsistentInitializationLinearBinding("LUMPED_RATE_MODEL_WITHOUT_PORES", 1e-12, 1e-14);
+	cadet::test::column::testTimeDerivativeJacobianFD("LUMPED_RATE_MODEL_WITHOUT_PORES");
 }
 
-TEST_CASE("LRM consistent initialization with SMA binding", "[LRM],[ConsistentInit]")
+TEST_CASE("LRM sensitivity Jacobians", "[LRM],[UnitOp],[Sensitivity],[CI]")
 {
-	std::vector<double> y(4 + 16 * (4 + 4), 0.0);
-// Optimal values:
-//	const double bindingCell[] = {1.2, 2.0, 1.0, 1.5, 858.034, 66.7896, 3.53273, 2.53153, 
-//		1.0, 1.8, 1.5, 1.6, 856.173, 64.457, 5.73227, 2.85286};
-	const double bindingCell[] = {1.2, 2.0, 1.0, 1.5, 840.0, 63.0, 3.0, 3.0, 
-		1.0, 1.8, 1.5, 1.6, 840.0, 63.0, 6.0, 3.0};
-	cadet::test::util::populate(y.data(), [](unsigned int idx) { return std::abs(std::sin(idx * 0.13)) + 1e-4; }, 4);
-	cadet::test::util::repeat(y.data() + 4, bindingCell, 16, 8);
-
-	cadet::test::column::testConsistentInitializationSMABinding("LUMPED_RATE_MODEL_WITHOUT_PORES", y.data(), 1e-14, 1e-5);
+	cadet::test::column::testFwdSensJacobians("LUMPED_RATE_MODEL_WITHOUT_PORES", 1e-4, 3e-7, 5e-5);
 }
 
-TEST_CASE("LRM consistent sensitivity initialization with linear binding", "[LRM],[ConsistentInit],[Sensitivity]")
+//TEST_CASE("LRM forward sensitivity vs FD", "[LRM],[Sensitivity],[Simulation],[failedFDtestLRM],[FD]") // todo fix tolerances
```

Comment:
Also here, please open an Issue and link.

Reply:
the corresponding issue would be the general fix tests issue that we have. I decided against opening an issue for all the things that are still missing here and instead have them documented in this PR

Reply:
Got it. Just note that on merging the PR, it will be closed and cannot be reopened.
For this purpose, I would suggest making a new (meta) issue "Testing: next steps" to collect all that is left to do after this (preliminary) work is finished.